inst/app.crunch.io/api/datasets/3.json

```diff
@@ -20,7 +20,8 @@
     },
     "fragments": {
         "table": "https://app.crunch.io/api/datasets/3/table/",
-        "preferences": "https://app.crunch.io/api/datasets/3/preferences/"
+        "preferences": "https://app.crunch.io/api/datasets/3/preferences/",
+        "pk": "https://app.crunch.io/api/datasets/3/pk/"
```

Comment:
For the dataset mockfile, I'm seeing just the key:value without the list syntax. There *is* a list in the actual pk.json response (both in my mock as well as what the api returns.